### DISCUSSION TITLE
how to change git language

### DISCUSSION THREAD

**Discussion by MoShangColdRain** [Homebrew/discussions#80] [Everyday usage]
now, my git language is chinese, but I don't like ,i want change to english. how to change it?

**Comment by Rylan12** [comment]
Try taking a look at [this discourse post](https://discourse.brew.sh/t/how-does-git-choose-its-language/4147) (specifically [this comment](https://discourse.brew.sh/t/how-does-git-choose-its-language/4147/5?u=rylan12)).

If that doesn't solve it, you may have better luck posting this question is the git forum as it seems this isn't really an issue with Homebrew.
